roles/confluent.variables/defaults/main.yml

```diff
@@ -58,6 +58,27 @@ common_role_completed: false
 
 proxy_env: {}
 
+### The method of installation. Valid values are "package" or "archive". If "archive" is selected then services will not be installed via the use of yum or apt, but will instead be installed via expanding the target .tar.gz file from the Confluent archive into the path defined by `archive_destination_path`. Configuration files are also kept in this directory structure instead of `/etc`. SystemD service units are copied from the ardhive for each target service and overrides are created pointing at the new paths. The "package" installation method is the default behavior that utilizes yum/apt.
+installation_method: "package"
+
+### The Confluent Platform version of the archive to download. Possible values: 5.5.0, 5.5.1, etc.
+confluent_archive_version: "{{ confluent_package_version }}"
+
+### The Scala version of the Confluent Platform archive to download. Possible values: 2.11, 2.12, etc. If you don't have a specific version requirement then use the default.
+confluent_archive_scala_version: 2.12
+
+### The path the downloaded archive is expanded into. Using the default with a `confluent_archive_version` of *5.5.1* results in the following installation path `/opt/confluent/confluent-5.5.1/` that contains directories such as `bin` and `share`, but may be overridden usinf the `binary_base_path` property.
+archive_destination_path: "/opt/confluent"
+
+### The base path for the configuration files. When in Archive File deployment mode this results in configuration files being based in `/opt/confluent/etc`, otherwise they are based in `/etc`.
+config_base_path: "{{ archive_destination_path if installation_method == 'archive' else '' }}"
+
+### The base path for the binary files. When in Archive File deployment mode this results in binary files being based in, for example `/opt/confluent/confluent-5.5.1/bin`, otherwise they are based in `/usr/bin`.
+binary_base_path: "{{ config_base_path+'/confluent-'+confluent_archive_version if installation_method == 'archive' else '/usr' }}"
```

Comment:
Do we want these be updatable?

I think no. Please move into `roles/confluent.variables/vars/main.yml'

Reply:
Done